packages/components/src/globals/scss/_feature-flags.scss

```diff
@@ -37,7 +37,7 @@ $default-feature-flags: (
 /// @access private
 /// @type Bool
 /// @group feature-flags
-$did-warn-diverged-feature-flags: false !default !global;
+$did-warn-diverged-feature-flags: false !default;
```

Comment:
`$did-warn-diverged-feature-flags` is declared at the global scope, so `!global` is redundant

Reply:
This is for:

```scss
.foo {
  @import 'carbon-components/scss/globals/scss/styles';
}
```

Given this is a flag to avoid extraneous warnings, the change is good, though.